src/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To do List</title>
+    <script src="https://kit.fontawesome.com/321e54ee43.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@400;500;600;700&family=Raleway:wght@300&display=swap" rel="stylesheet">
+    </head>
+<body>
+    <div class="logo">Carrera<i class="fa-solid fa-circle-dot"></i>Programmer</div>
```

Comment:
Kindly. I recommend putting all tags inside of <section>. Your code will be more tidy and clean